gitops/snapshot.go

```diff
@@ -117,6 +117,25 @@ const (
 	AppStudioIntegrationStatusFinished = "Finished"
 )
 
+// IntegrationTestScenario test runs status
+type TestStatus int
+
+//go:generate enumer -type=TestStatus -linecomment -json
```

Comment:
The usage looks great. I didn't know it before, let me search a doc to learn. Thanks